module/programs/helper.js

```diff
@@ -870,20 +861,30 @@ module.exports = class ProgramsHelper {
             });
             scopeData = _.omit(scopeData, keysCannotBeAdded);
           }
+
+          let tenantDetails = await userService.fetchPublicTenantDetails(tenantData.tenantId);
+          if (!tenantDetails.data || !tenantDetails.data.meta || tenantDetails.success !== true) {
```

Comment:
@borkarsaish65 this also can be optimized like above comment and send with error code